micmac_li3ds/import_autocal.py

```diff
@@ -62,158 +59,174 @@ def get_parser(self, prog_name):
             '--indent', type=int,
             help='number of spaces for pretty print indenting')
         parser.add_argument(
-            'filename',
+            'filename', nargs='+',
             help='the autocal file')
```

Comment:
What about mentioning in the help message that multiple files may be specified on the command line?